--- org/jivesoftware/smack/provider/ProviderManager.java
+++ org/jivesoftware/smack/provider/ProviderManager.java
@@ -158,7 +158,7 @@
             ClassLoader[] classLoaders = getClassLoaders();
             for (ClassLoader classLoader : classLoaders) {
                 Enumeration providerEnum = classLoader.getResources(
-                        "META-INF/smack.providers");
+                        "res/raw/smack.providers");
                 while (providerEnum.hasMoreElements()) {
                     URL url = (URL) providerEnum.nextElement();
                     InputStream providerStream = null;
